docs/5.x/terraform.md

```diff
@@ -0,0 +1,534 @@
+# Terraform Provider (OSS)
+
+The gravity terraform provider is used to support terraform management of opensource gravity clusters. The provider needs to be configured with a valid token in order to manage a cluster.
+
+## Getting Started
+
+### Install the Gravity provider
+The terraform provider will be automatically installed when getting the gravity tools.
+
+```
+curl https://get.gravitational.io/telekube/install/5.0.23 | bash
```

Comment:
I would either just redirect to "getting the tools" section so we don't duplicate this, or gave a curl | bash command to download only the tf provider binary and not everything.